configure.ac

```diff
@@ -269,4 +294,13 @@ AC_MSG_RESULT([
                To enable this feature please install libdaxctl-devel v66 or later
                and run configure again])
 fi
+if test $hwloc_hmat = "yes" ; then
+AC_MSG_RESULT([
+               Utilize information about memory performance characteristics is enabled])
```

Comment:
Please correct: Utilization of information

Reply:
Done in #507 ( after merge #507 it will be available in this PR)